### PR TITLE
feat(maintenance): add sync network and change password tasks

### PR DESCRIPTION
**Pull request by tim-shockley**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,33 @@
-# RPC Beta
+# Remote Provisioning Client (RPC) Beta
 
 > Disclaimer: Production viable releases are tagged and listed under 'Releases'. All other check-ins should be considered 'in-development' and should not be used in production
 
+Used for activation, deactivation, maintenance, and status of AMT
```
Comment (rsdmike) [inline README.md:5]: This is good info, but we keep this kind of info out of the readme and keep it in the docs. Can you move it to https://open-amt-cloud-toolkit.github.io/docs/2.5/Reference/RPC/commandsRPC/#maintenance, or ask @bwendlandt-intel  for help and he can do it.

File: flags.go [hunk 2]
```diff
@@ -17,28 +20,71 @@ import (
...
+	}
+	return *a.val
+}
+func (a *IpVar) Set(val string) error {
+	if net.ParseIP(val) == nil {
+		return errors.New("not a valid ip address")
+	}
+	*a.val = val
+	return nil
+}
```
Comment (rsdmike) [inline flags.go:57]: why not just a single function that checks if the IP is valid? This seems a bit heavy?

File: flags.go [hunk 3]
```diff
@@ -116,38 +188,126 @@ func (f *Flags) setupCommonFlags() {
...
+
+	ifaces, err := f.netEnumerator.Interfaces()
+	if err != nil {
+		log.Error(err)
+		return ""
+	}
+
+	for _, i := range ifaces {
+		// this handles the check for command line param and the first matched lookup value
+		if len(f.IpConfiguration.IpAddress) != 0 {
```
Comment (rsdmike) [inline flags.go:267]: If they've provided an IP address, I think skip getting the lan interface settings entirely and return up at line 253.

**Review by rsdmike — CHANGES_REQUESTED** [review]

File: flags.go (hunk 2 above)
Comment (tim-shockley) [reply flags.go]: Wanted validation to be handled by the library with the variable registration and that was the best example implementation I could find at the time and my library/golang expertise could grok. Another search with more experience, found a better approach.

File: flags.go (hunk 3 above)
Comment (tim-shockley) [reply flags.go:264]: done

**Review by rsdmike — APPROVED** [review]

**Pull request merged**